src/Kunstmaan/NodeBundle/Controller/WidgetsController.php

```diff
@@ -53,40 +53,6 @@ public function selectLinkAction(Request $request)
         return $params;
     }
 
-    /**
```

Comment:
Why is this method being removed? Does the urlchooser still work after this?

Reply:
@sandergo90 said these methods were not used and could be removed. But I will check later if everything still works!

Reply:
@Numkil this method is never used, if you check the entire code base, there are no usages of this. The URL chooser also used another action.